client/core/core.go

```diff
@@ -5630,6 +5630,29 @@ func (c *Core) PreOrder(form *TradeForm) (*OrderEstimate, error) {
 	}, nil
 }
 
+// MultiTrade is used to place multiple standing limit orders on the same
+// side of the same market simultaneously.
+func (c *Core) MultiTrade(pw []byte, form *MultiTradeForm) ([]*Order, error) {
+	reqs, err := c.prepareMultiTradeRequests(pw, form)
+	if err != nil {
+		return nil, err
+	}
+
+	orders := make([]*Order, 0, len(reqs))
+
+	var numSuccessful uint64
+	for _, req := range reqs {
+		corder, err := c.sendTradeRequest(req)
+		if err != nil {
+			return nil, fmt.Errorf("error sending trade request after %d successful sends: %v", numSuccessful, err)
```

Comment:
This is dicey because they don't know what the orders are they just submitted if some succeeded.
One kinda wonky pattern I've seen is to return a custom error type that includes any data, which could be the partial list of submitted orders.

```go
type errorMultiTrade struct {
    error
    submitted []*Order
}
```

Any better ways to deal with this?